### ISSUE TITLE
Saving issues : "missing chunks" 

### ISSUE THREAD

**Issue by parandar** [maruohon/litematica#70]
Hello, using Litematica for 1.14.4 (Fabric) version dev20190720 found on curseforge, with maLiLib 0.10.0-dev20. 

When I try to save a selection I get a message bottom right of my screen saying :
`Saving schematic, missing chunks (1)`

And so no matter the selection. 
No file got created. Operation still in task manager, but no progress even after a while.

(even if I can't save my selection, nice work on that mod!)

**Comment by vacla** [comment]
fly/move around to load the chunks, does that work?

**Comment by maruohon** [comment]
Did you move close to the chunk it says it's waiting for? Did you accidentally forget a sub-region box somewhere else far away? (You can see the sub-regions for the current selection in the Area Editor GUI.)

**Comment by parandar** [comment]
I was like 10 block away, then went precisely on the spot for no visible change. 
Went far away and back again and no change as well.

I doubled checked for sub region and haven't got a single one elsewhere. I just literally opened a world to test it right on the spot. Didn't really move around.

I tried your dev version : 0.0.0-dev.20190807.003724
No change if I'm trying the same procedure.

I'm going to try to launch java in Administrator if it's a problem regarding the file creation, but I doubt it
